surface/model_openapiv3.go

```diff
@@ -156,10 +199,15 @@ func (b *OpenAPI3Builder) buildMethodFromPathItem(
 func (b *OpenAPI3Builder) buildTypeFromParameters(
 	name string,
 	parameters []*openapiv3.ParameterOrReference,
-	requestBody *openapiv3.RequestBodyOrReference) (typeName string, err error) {
+	requestBody *openapiv3.RequestBodyOrReference,
+	fromComponent bool) (typeName string, err error) {
 	t := &Type{}
 	t.Name = name + "Parameters"
 	t.Description = t.Name + " holds parameters to " + name
+	if fromComponent {
```

Comment:
can we put this code following the rest of the default initialization of `t`? Just to avoid breaking it up. `if fromComponent` is a "special" case and it reduces readability to drop that in the middle of default init of a struct

```go
t := &Type{}
t.Name = name + "Responses"
t.Description = t.Name + " holds responses of " + name
t.Kind = TypeKind_STRUCT
t.Fields = make([]*Field, 0)

if fromComponent { 
...
```

Reply:
Done